src/debugger/osx_ssh.md

```diff
@@ -0,0 +1,16 @@
+# Debugging on MacOS over SSH
+
+If you try to debug a program over SSH, you may experience failures or rizin
+getting stuck while opening the file. This is because the OS is waiting for
+user authentication before allowing the debugging, however since you are over
+SSH the OS has no way of showing the permission window.
+
+To avoid this problem you can either run rizin with `sudo` or you may instruct taskport to not authenticate the user. To do so you can do:
+
+```sh
+security authorizationdb read system.privilege.taskport > taskport.plist
+/usr/libexec/PlistBuddy -c 'Set :authenticate-user false' ./taskport.plist
+sudo security authorizationdb write system.privilege.taskport < taskport.plist
+```
+
+The above will disable the debugging authentication prompt even after you reboot.
```

Comment:
I would explain it before showing the commands. First describe what will happen, and then share how